tests/test_examples.py

```diff
@@ -103,3 +103,28 @@ def test_tracking_merlion(reset_database, engine: sa.Engine, tracking_store: Sql
         assert session.query(SqlExperiment).count() == 2
         assert session.query(SqlMetric).count() == 4
         assert session.query(SqlParam).count() == 5
+
+    with engine.begin() as conn:
+        conn.execute(sa.sql.text("REFRESH TABLE doc.anomalies"))
+        conn.execute(sa.sql.text("REFRESH TABLE testdrive.runs"))
```

Comment:
Just calling out to `sa.text()` should work.
```suggestion
        conn.execute(sa.text("REFRESH TABLE doc.anomalies"))
        conn.execute(sa.text("REFRESH TABLE testdrive.runs"))
```